spec/use_cases/administrator/get_auth_requests_for_username_spec.rb

```diff
@@ -0,0 +1,43 @@
+describe UseCases::Administrator::GetAuthRequestsForUsername do
+  let(:authentication_logs_gateway) { Gateways::LoggingApi.new }
+  let(:username) { "AAAAA" }
+  let(:logs) do
+    [
+      {
+        "ap" => "",
+        "building_identifier" => "",
+        "id" => 1,
+        "mac" => "",
+        "siteIP" => "",
+        "start" => "2018-10-01 18:18:09 +0000",
+        "stop" => nil,
+        "success" => true,
+        "username" => username
+      },
+      {
+        "ap" => "",
+        "building_identifier" => "",
+        "id" => 1,
+        "mac" => "",
+        "siteIP" => "",
+        "start" => "2018-10-01 18:18:09 +0000",
+        "stop" => nil,
+        "success" => true,
+        "username" => username
+      }
+    ]
+  end
+
+  subject { described_class.new(authentication_logs_gateway: authentication_logs_gateway) }
```

Comment:
Wonder if we should send in a dummy gateway, stubbing the real gateway feels like we are not reaping the benefits of the Clean Architecture approach.

Reply:
🐺